### PR TITLE
Ar adm 53 lesson series by tag

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/shared/Fields/Source.js [hunk 1]
```diff
@@ -25,9 +26,9 @@ class SourceField extends PureComponent {
     source: null,
   };
 
-  handleSelect = (source) => {
-    this.props.onChange(this.props.getSourceById(source.id));
-  };
+  handleSelect = (source) => this.props.onChange(this.props.getSourceById(source.id));
+
+  handleRemove = () => this.props.onChange({ id: null });
```
Comment (edoshor) [inline src/components/shared/Fields/Source.js:31]: pass a simple null to `onChange` not a crippled object. Let the caller handle it.

File: src/components/shared/Forms/Collection/BaseCollectionForm.js [hunk 2]
```diff
@@ -182,10 +184,16 @@ class BaseCollectionForm extends Component {
 
   handleSourceChange = (source) => {
     const { errors } = this.state;
-    delete errors.source;
+    delete errors.series;
```
Comment (edoshor) [inline src/components/shared/Forms/Collection/BaseCollectionForm.js:187]: What is `series` ?
keep `source`

File: src/components/shared/Forms/Collection/BaseCollectionForm.js [hunk 3]
```diff
@@ -200,8 +208,9 @@ class BaseCollectionForm extends Component {
       return;
     }
 
-    const properties = cleanProperties(this.getPropertiesFromState());
-    const i18n       = this.cleanI18n();
+    const properties  = cleanProperties(this.getPropertiesFromState());
+    const i18n        = this.cleanI18n();
+    properties.source = this.state.source || '';
```
Comment (edoshor) [inline src/components/shared/Forms/Collection/BaseCollectionForm.js:213]: no need to have source as key in all properties on all collections.
We only need it in specific types of collection.

File: src/components/shared/Fields/Source.js (hunk 1 above)
Comment (davgur) [reply src/components/shared/Fields/Source.js]: done

File: src/components/shared/Forms/Collection/BaseCollectionForm.js (hunk 3 above)
Comment (davgur) [reply src/components/shared/Forms/Collection/BaseCollectionForm.js]: add If

File: src/components/shared/Forms/Collection/BaseCollectionForm.js (hunk 2 above)
Comment (davgur) [reply src/components/shared/Forms/Collection/BaseCollectionForm.js]: I use same error for tags and source (one of this must be required). remane to sourceOrTag

**Review by edoshor — APPROVED** [review]

**Pull request merged**